### DISCUSSION TITLE
support for Read/Write for HF and LF

### DISCUSSION THREAD

**Discussion by gitupsidewild** [wh201906/Proxmark3GUI#45] [General]
HF seems to have more support at the moment why not have features added to LF?

**Comment by wh201906** [comment]
I designed this GUI as an alternative of many Chinese paid GUIs, which also focus on HF(mostly mifare classic) and have little support on LF. It covers most of my requirements so I don't put new features on it. Plus, when you get deeper into the Proxmark3, you will find the CLI is always the most powerful one.
Having said that, I might implement more features in the future, if I have enough time for this.

**Comment by wh201906** [comment]
If you are looking for a general purpose GUI, you can try this
https://github.com/burma69/PM3UniversalGUI
